devices/src/legacy/i8042.rs

```diff
@@ -63,8 +63,7 @@ const SB_I8042_CMD_DATA: u8 = 1 << 3; // i8042 expecting command parameter at po
 const SB_KBD_ENABLED: u8 = 1 << 4; // 1 = kbd enabled, 0 = kbd locked
 
 /// i8042 control register bits
-#[allow(clippy::identity_op)]
-const CB_KBD_INT: u8 = 1 << 0; // kbd interrupt enabled
+const CB_KBD_INT: u8 = 1; // kbd interrupt enabled
```

Comment:
clippy is being stupid here and 1 << 0 is more expressive in this case.

Reply:
Okay sure, I will revert this -- but to optimize the number of CI calls I'll do a push when I get some more feedback

Reply:
Hmmm? What do you mean @petreeftime? Clippy is recommanding to **not** use 1 << 0 and just use 1 instead.

Reply:
Agreed. But in this particular case this value means `bit 0`, which is better expressed as `1 << 0`. `1` by itself loses that meaning.

Reply:
It's also inconsistent with the rest of the  `const`s in this file, which have similar meanings.

Reply:
If we do want to be consistent then we should define these flags in hex. That's also how they're defined in the kernel and in the key scan codes (below in the code).

How about his solution?

Reply:
https://elixir.bootlin.com/linux/latest/source/include/linux/i8042.h#L47

Reply:
Looks good!

Reply:
Okay I will change them to hex